packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # prosekit
 
+## 0.22.0-beta.19
+
+### Patch Changes
+
+- [`e6747b3`](https://github.com/ocavue/prosekit/commit/e6747b3ead70e85fb46ce53e021c03d6705c3d80) ![](https://prosekit.dev/b/extensions)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)